AppCenterAuth/AppCenterAuth/Internals/Util/MSAuthConstants.h

```diff
@@ -25,3 +25,13 @@ static NSString *const kMSAuthConfigFilename = @"config.json";
  * The eTag key to store the eTag of current configuration.
  */
 static NSString *const kMSAuthETagKey = @"MSAuthETagKey";
+
+/**
+ * B2C authority type
+ */
+static NSString *const kMSAuthorityTypeB2C = @"B2C";
```

Comment:
Those constants are very specific to AAD and B2C authorities. I would suggest to move this to each authority class and remove the utility method in the base class. You can check `type` in a dictionary in `MSAuthConfig` at the time it instantiate an object.

Reply:
Got it. But if check type in MSAuthConfig. I would require those constants again in this file. Can you please suggest where should I put these constants so that they are accessible to all files in a folder?

Reply:
You can define those in each class and reference it from there.

Reply:
Move this to `MSB2CAuthority.h`.

Reply:
This is also used by MSAuthConfig to instantiate object of type of authority.